x-pack/plugins/session_view/public/components/detail_panel_process_tab/styles.ts

```diff
@@ -27,10 +27,37 @@ export const useStyles = () => {
       textOverflow: 'ellipsis',
     };
 
+    const metadataHostOS: CSSObject = {
+      margin: `${euiTheme.size.m} ${euiTheme.size.base} ${euiTheme.size.base} ${euiTheme.size.base}`,
+      paddingBottom: euiTheme.size.base,
+    };
+
+    const accordion: CSSObject = {
```

Comment:
I think we shouldn't need these accodion styles if we use `DetailPanelAccordion` above